src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsConformingFractures.cpp

```diff
@@ -714,6 +725,23 @@ void SinglePhasePoromechanicsConformingFractures::updateState( DomainPartition &
   Base::updateState( domain );
 
   updateHydraulicApertureAndFracturePermeability( domain );
+
+  forDiscretizationOnMeshTargets( domain.getMeshBodies(), [&] ( string const &,
+                                                                MeshLevel & mesh,
+                                                                arrayView1d< string const > const & regionNames )
+  {
+    ElementRegionManager & elemManager = mesh.getElemManager();
+
+    elemManager.forElementSubRegions< FaceElementSubRegion >( regionNames,
+                                                              [&]( localIndex const,
+                                                                   FaceElementSubRegion & subRegion )
+    {
+      // update fluid model
+      poromechanicsSolver()->flowSolver()->updateFluidState( subRegion );
+      // update solid internal energy
+      poromechanicsSolver()->flowSolver()->updateSolidInternalEnergyModel( subRegion );
```

Comment:
Is it going to work if the problem is isothermal and there is no solid internal energy model registered?

Reply:
probably not, I will add a check.